optics/tests/Optics/Tests/Core.hs

```diff
@@ -92,7 +92,7 @@ coreTests = testGroup "Core"
   , testCase "optimized rhs13" $
     assertSuccess $(inspectTest $ hasNoProfunctors 'rhs13)
   , testCase "traverseOf_ itraversed = traverseOf_ folded" $
-    assertSuccess $(inspectTest $ 'lhs14 ==- 'rhs14)
+    ghc92failure $(inspectTest $ 'lhs14 ==- 'rhs14)
```

Comment:
Should there be a comment why this fails?

Other seems to be known issues which is reassuring

Reply:
I added a comment, it's a similar issue to others.